From e64002cdb41fdb8e4f529fc0e857f86bc61d40ba Mon Sep 17 00:00:00 2001
From: Bai Ping <ping.bai@nxp.com>
Date: Thu, 16 Nov 2017 10:21:57 +0800
Subject: [PATCH 2805/5242] MLK-16829 ARM64: dts: remove unused cpu idle state
 on imx8mq

commit  0572eb1946db2863e76511e82feb905c69447be5 from
https://source.codeaurora.org/external/imx/linux-imx.git

The cluster level can NOT be power down on i.MX8MQ, so the cluster
level idle state can be removed. Only core level power down is used
for cpuidle support.

Signed-off-by: Bai Ping <ping.bai@nxp.com>
Reviewed-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi |   23 +----------------------
 1 file changed, 1 insertion(+), 22 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
index 964abaf..b54013b 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
@@ -48,15 +48,7 @@
 
 			CPU_SLEEP: cpu-sleep {
 				compatible = "arm,idle-state";
-				arm,psci-suspend-param = <0x0000000>;
-				entry-latency-us = <700>;
-				exit-latency-us = <250>;
-				min-residency-us = <1000>;
-			};
-
-			CLUSTER_SLEEP: cluster-sleep {
-				compatible = "arm,idle-state";
-				arm,psci-suspend-param = <0x1010033>;
+				arm,psci-suspend-param = <0x0010033>;
 				local-timer-stop;
 				entry-latency-us = <1000>;
 				exit-latency-us = <700>;
@@ -1190,7 +1182,6 @@
 };
 
 &A53_0 {
-	cpu-idle-states = <&CPU_SLEEP &CLUSTER_SLEEP>;
 	operating-points = <
 		/* kHz    uV */
 		1000000 900000
@@ -1204,15 +1195,3 @@
 	clock-latency = <61036>;
 	#cooling-cells = <2>;
 };
-
-&A53_1 {
-	cpu-idle-states = <&CPU_SLEEP &CLUSTER_SLEEP>;
-};
-
-&A53_2 {
-	cpu-idle-states = <&CPU_SLEEP &CLUSTER_SLEEP>;
-};
-
-&A53_3 {
-	cpu-idle-states = <&CPU_SLEEP &CLUSTER_SLEEP>;
-};
-- 
1.7.9.5

